wolox-books/src/app/screens/modal-car/modal-car.component.scss

```diff
@@ -0,0 +1,31 @@
+@import 'src/scss/color';
+@import 'src/scss/fonts';
+
+.content-books-car {
+  margin: 30px 0;
+  position: relative;
+
+  display: flex;
+    justify-content: space-between;
+
```

Comment:
You should add align-items center, if you see the screenshot in the pr the information is not vertically centered